site/app/controllers/GlobalController.php

```diff
@@ -64,6 +64,15 @@ public function header() {
                     "icon" => "fa-star"
                 ]);
             }
+            elseif ($this->core->getUser()->getAccessLevel() === User::LEVEL_FACULTY) {
```

Comment:
Please allow superusers to also create couses with the interface
(access level should set minimum requirements)